src/index.css

```diff
@@ -1,27 +1,43 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+.special-button {
+  background-color: orange;
+}
+
+#root {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 10px;
+  width: 400px;
+}
+
+#root .view-num {
+  background-color: #a1a1a1;
+  width: 100%;
+  padding: 15px 5px;
```

Comment:
[ OPTIONAL]
- [ ] You may consider sticking to the wireframe by using the same exact color.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/204538504-30e00118-3d74-42c0-8e39-994b2633768e.png)
![image](https://user-images.githubusercontent.com/97100091/204538638-96058fed-cf50-47d1-ae16-6f5c9e81f399.png)

template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/204538425-490db1cc-1f30-4262-babc-5d2afa1be813.png)
![image](https://user-images.githubusercontent.com/97100091/204538691-cd7dd139-f18f-4642-8d1c-72d0cd8403cf.png)